simconvoi.h

```diff
@@ -229,6 +229,15 @@ class convoi_t : public sync_steppable, public overtaker_t
 	*/
 	convoihandle_t coupling_convoi;
 
+	/**
+	* a convoy that is coupling now.
+	*/
+	convoihandle_t will_coupling_convoi;
```

Comment:
"in progress"の単語の方が実態に近いかなと思います。
```suggestion
	convoihandle_t convoi_coupling_in_progress;
```